### PR TITLE
Fix formatting of code samples

### PR DESCRIPTION
**Pull request by dksimpson**
Story [1510854](https://mseng.visualstudio.com/TechnicalContent/_workitems/edit/1510854/)
Content Update - Localization fix: CC122748: Updated bad code formatting and trailing whitespace

### CODE REVIEW

File: docs/framework/wcf/feature-details/interoperable-object-references.md [hunk 1]
```diff
@@ -48,43 +48,34 @@ public class SomeClass
...
- `public SomeClass B = someInstance;`  
-  
- `}`  
-  
- `public class SomeClass`  
-  
- `{`  
-  
- `}`  
+```
```
Comment (NextTurn) [inline docs/framework/wcf/feature-details/interoperable-object-references.md]:
```suggestion
```csharp
```

File: docs/framework/wcf/feature-details/interoperable-object-references.md [hunk 2]
```diff
@@ -48,43 +48,34 @@ public class SomeClass
...
+[DataContract(IsReference=true)] public class X`
```
Comment (NextTurn) [inline docs/framework/wcf/feature-details/interoperable-object-references.md]:
```suggestion
[DataContract(IsReference=true)]
public class X  
```

File: docs/framework/wcf/feature-details/interoperable-object-references.md [hunk 3]
```diff
@@ -48,43 +48,34 @@ public class SomeClass
...
+}
+  
+public class SomeClass 
+{   
+}  
+````
+
  The generated XML is as follows:  
+
+```
```
Comment (NextTurn) [inline docs/framework/wcf/feature-details/interoperable-object-references.md]:
```suggestion
```xml
```

File: docs/framework/wcf/feature-details/interoperable-object-references.md [hunk 4]
```diff
@@ -48,43 +48,34 @@ public class SomeClass
...
- `<Value>contents of A</Value>`  
-  
- `</A>`  
-  
- `<B ref="1">`  
-  
- `</B>`  
-  
- `</X>`  
+    <B ref="1">
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/interoperable-object-references.md]:
```suggestion
    <B ref="1"></B>
```

File: docs/framework/wcf/feature-details/interoperable-object-references.md [hunk 5]
```diff
@@ -48,43 +48,34 @@ public class SomeClass
...
+    </B>
```
Comment (rpetrusha) [inline docs/framework/wcf/feature-details/interoperable-object-references.md]:
```suggestion
 ```

**Comment by Thraka** [comment]
@dksimpson The suggestions from @NextTurn are good. Please accept them. Thanks!

**Review by rpetrusha — CHANGES_REQUESTED** [review]
Thanks for correcting this badly formatted code, @dksimpson. Both @NextTurn and I have left some suggested changes.

**Comment by dksimpson** [comment]
@NextTurn @rpetrusha @Thraka  Thanks everyone for your review comments - I've updated the article.

**Review by rpetrusha — APPROVED** [review]

**Comment by rpetrusha** [comment]
Thanks, @dksimpson, for making the additional changes. I'll merge your PR now.

**Pull request merged**